### ISSUE TITLE
Gravitas a bit too overpowerd?

### ISSUE THREAD

**Issue by IdkSomethingTemporary** [Arch-Shaman/ZK-Futurewars-Mod#458]
**Unit**: Gravitas / Jugglenaut / Jumpsumo
**Problem**: It's just a bit too overpowered in general. The best unit to compare it to would probably be colossus, both of which fill a similar niche of being a tanky frontliner with low dps but high durability. However, simply looking at the stats one sees a problem

1. Price: Gravitias is close to 2/3rd the cost of Colossus, 4350 vs 6000, **however, this difference in price is not reflected sufficiently in the difference in power**
2. Health: Gravitas has more health than Colossus, but Colossus has a much higher autoregen. Overall, Colossus probably will have more effective health thank's to it's regen, especially considering it allows colossus to operate without a fragile repair crew.
3. Damage: Gravitas has slightly less raw dps than Colossus, 500 dps vs 650, however, Gravitias has a _much_ higher splash radius than Colossus, so overall the damage is about the same, if not higher
4. Dgun: Both striders have a pretty similarly effective Dgun. Both have a dgun which is able to provide lots of crow control damage. I haven't used Colossus' dgun much or seen it used much, but on paper it simply has less range and ultility than Gravitas' Jump
5. Ultility: Gravitas is able to control the enemy with it's gravity effect. This allows it to engage less enemies at once, or to grab a high value escaping target. Colossus does not have this ultility

TL:DR - Gravitas needs a nerf, prehaps not right now, but after it enters mainstream use, but either way, it needs a nerf

**Comment by Arch-Shaman** [comment]
`Dgun: Both striders have a pretty similarly effective Dgun. Both have a dgun which is able to provide lots of crow control damage. I haven't used Colossus' dgun much or seen it used much, but on paper it simply has less range and ultility than Gravitas' Jump`

Colossus dgun can be used on a slight incline to become an artillery piece. Though I suppose this should be investigated as to why people aren't using the dgun.

**Comment by IdkSomethingTemporary** [comment]
Looks like there hasn't been complaints in half a year, probably good to close
